### ISSUE TITLE
Number + `=` does not increase title level correctly in markdown

### ISSUE THREAD

**Issue by newptcai** [vimwiki/vimwiki#1270]
If I write
```
# Test
```
and press `3=` or `4=`, etc., the level of the title is only increased by two.

I am using markdown format. My `.vimrc` is the following

```
" Temporary wiki
let g:vimwiki_global_ext = 1

"let g:vimwiki_list = [{'path': '~/vimwiki',
"                    \ 'syntax': 'markdown', 'ext': '.md', 'auto_tags': 1}]

let g:vimwiki_folding = 'expr'
let g:vimwiki_ext2syntax = {'.md': 'markdown',
                \ '.mkd': 'markdown',
                \ '.wiki': 'media'}
" Auto save
let g:vimwiki_autowriteall=0

" -----------------------------------------------------------------------------
" To make it work with UltilSnips
" See: https://github.com/vimwiki/vimwiki/issues/357#issuecomment-739437349
" -----------------------------------------------------------------------------
let g:vimwiki_key_mappings = { 'table_mappings': 0, }

" calendar
let g:vimwiki_use_calendar = 1
```

**Comment by tinmarino** [comment]
My bad at 5e564bb10dc52 should be fixed now in the dev branch since 5db16850da59015.
@newptcai can you please check and close?

**Comment by newptcai** [comment]
This one works. Thanks.
